app/controllers/calls_controller.rb

```diff
@@ -88,7 +88,12 @@ def render_twiml(response)
 
   def active_call
     remote_id = params['CallSid'] || params[:remote_id]
-    @call = Ivr::Call.includes(connections: :legislator).where(remote_id: remote_id).first_or_create
+    @call = Ivr::Call.includes(connections: :legislator).first_or_initialize(remote_id: remote_id)
+    if @call.new_record?
+      @call.person ||= Person.find_or_create_by(phone: params[:From])
```

Comment:
need to change email validation on Person if you want to `create_by(phone: phone)`